handlers_test.go

```diff
@@ -30,8 +32,9 @@ func TestHandlers(t *testing.T) {
 		{"Not found - get organisation by uuid", newRequest("GET", fmt.Sprintf("/transformers/organisations/%s", testUUID)), &dummyService{found: false, initialised: true, orgs: []org{org{}}}, http.StatusNotFound, "application/json", ""},
 		{"Service unavailable - get organisation by uuid", newRequest("GET", fmt.Sprintf("/transformers/organisations/%s", testUUID)), &dummyService{found: false, initialised: false, orgs: []org{}}, http.StatusServiceUnavailable, "application/json", ""},
 		{"Success - get organisations", newRequest("GET", "/transformers/organisations"), &dummyService{found: true, initialised: true, orgs: []org{org{UUID: testUUID}}}, http.StatusOK, "application/json", getOrganisationsResponse},
-		{"Not found - get organisations", newRequest("GET", "/transformers/organisations"), &dummyService{found: false, initialised: true, orgs: []org{}}, http.StatusNotFound, "application/json", ""},
 		{"Service unavailable - get organisations", newRequest("GET", "/transformers/organisations"), &dummyService{found: false, initialised: false, orgs: []org{}}, http.StatusServiceUnavailable, "application/json", ""},
+		{"Success - get count", newRequest("GET", "/transformers/organisations/__count"), &dummyService{found: true, initialised: true, orgs: []org{org{UUID: testUUID}}}, http.StatusOK, "application/json", "1\n"},
```

Comment:
This will break the concept-publisher as it expects plain test without a new line at the end :(